workitem/link/link_repository_blackbox_test.go

```diff
@@ -61,6 +63,104 @@ func (s *linkRepoBlackBoxTest) TestList() {
 	})
 }
 
+func (s *linkRepoBlackBoxTest) TestChildrenOrderOfExecution() {
```

Comment:
This test can be removed as it is not setting the order explicitly. The other tests looks good.

Reply:
ok @baijum

Reply:
removed the test here 2f7e42c